tough/src/lib.rs

```diff
@@ -198,72 +231,76 @@ pub struct Repository {
 }
 
 impl Repository {
+    /// Load and verify TUF repository metadata with default [`Options`]. See
+    /// [`Repository::load`] for more into.
+    pub fn load_default<R, S1, S2>(settings: Settings<R, S1, S2>) -> Result<Self>
```

Comment:
I expect it to be pretty rare that people want to customize the options.  Should this be `load` and `load` be `load_custom` or whatever?  (If we change or eliminate the settings / options split, this wouldn't apply, of course.)

Reply:
Yeah - I like changing this to `load`.  So far we haven't needed to change options in any of our code.

Reply:
My IDE made this possible in 2 seconds, yay. I agree this is better.
fixup: simplify repository load - load_with_options

Reply:
Addressed in [59899dd](https://github.com/awslabs/tough/compare/3e858e5b5d629576a0c13f3f7b627cd8c49af9a4..59899ddf735ceb7e4edc883a9b1c6a195506f820),